template/php/function/src/Handler.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App;
+
+/**
+ * Class Handler
+ * @package App
+ */
+class Handler
+{
+    /**
+     * @param string $data
+     * @return string
+     */
+    public function handle(string $data): string {
```

Comment:
Is a string going to be suitable for everyone?

Reply:
It'd be nice to be an array, or abstracted to an object where the request body, as well as any headers etc could be accessed easily.

Reply:
This is a good point, this would prevent binary data being returned (or accepted). This requirements could be removed and be left to the application to handle.

Reply:
Can you provide an alternative option as a comment here which retains the simplicity? I'd like to take a look

Reply:
As a simple solution would adding a comment with the binary return type example?

Reply:
Scratch that, wouldn't work... Unless we alter the `new` command to take arguments, `new --ascii`, `new --json`, `new --binary` etc. Hackiet option would be having those flags as suffixes to different template files

Reply:
@alexellis wdyt? ^

Reply:
It's defining something I guess that's forcing type, I think the application should be more fluid because of the multi-use of the function. In theory, a custom class could be made and then the subsequent function could ask for the type based on that but, that's adding a layer of "custom code" on top. I think maybe leaving this open and not enforcing a type (as to, leave it to the application) would be a better option?